### PR TITLE
Add Web Audio API status

### PR DESCRIPTION
**Pull request by neuhaus**
Web Audio API is required for 3d audio which is essential for VR

### CODE REVIEW

File: src/data.json [hunk 1]
```diff
@@ -323,6 +323,41 @@
       "samsung": {
         "supported": 0
       }
+    },
+    {
+      "name": "Web Audio API",
+      "description": "positional audio is essential for VR.<br><br><a href=\"https://webaudio.github.io/web-audio-api/#Spatialization\">Spec</a>.",
```
Comment (cvan) [inline src/data.json]: nitpick, but the Web Audio API is useful in a (Web)VR context for more than only positional audio, though that's a primary use case

File: src/data.json [hunk 2]
```diff
@@ -323,6 +323,41 @@
...
+      "chrome": {
+        "icon": "chromium",
+        "supported": 1,
+        "minVersion": 49
+      },
+      "firefox": {
+        "icon": "firefox-nightly",
```
Comment (cvan) [inline src/data.json]: can you remove this line?

File: src/data.json [hunk 3]
```diff
@@ -323,6 +323,41 @@
       "samsung": {
         "supported": 0
       }
+    },
+    {
+      "name": "Web Audio API",
+      "description": "positional audio is essential for VR.<br><br><a href=\"https://webaudio.github.io/web-audio-api/#Spatialization\">Spec</a>.",
+      "chrome": {
+        "icon": "chromium",
```
Comment (cvan) [inline src/data.json]: can you remove this line?

File: src/data.json (hunk 1 above)
Comment (cvan) [inline src/data.json]: while you're in there, could you also capitalise the first sentence?

Comment (cvan) [inline src/data.json]: I think it's fine to keep the section name (the [`"name"` key above](https://github.com/Web-VR/iswebvrready/pull/92/files#diff-ce25d95e59175e2ecedf53bc48699377R328)) listed as the `Web Audio` and just change the text of this `"description" key` to mention `positional audio` (you can put it in parentheses), like so:

> "description": "Necessary for manipulating audio on the Web, allowing developers to choose audio sources, add effects to audio, create audio visualizations, apply spatial effects (e.g., panning), etc.<br><br><a href=\"https://webaudio.github.io/web-audio-api/\">Spec</a>. <a href=\"https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API\">Docs</a>."

File: src/data.json [hunk 4]
```diff
@@ -323,6 +323,41 @@
...
+        "supported": 1,
+        "minVersion": 49
```
Comment (cvan) [inline src/data.json]: nice job getting these version numbers. I assume these are specific to Web Audio Spatialization, but in any case, can we change these numbers to match the non-vendor-prefixed versions listed on [`Can I Use?`](http://caniuse.com/#feat=audio-api)?

File: src/data.json (hunk 1 above)
Comment (neuhaus) [inline src/data.json]: OK, I wasn't aware and I'm option to suggestions

File: src/data.json (hunk 2 above)
Comment (neuhaus) [inline src/data.json]: done

File: src/data.json [hunk 5]
```diff
@@ -323,6 +323,41 @@
...
+        "minVersion": 47
+      },
+      "opera": {
+        "supported": 1,
+        "minVersion": 39
+      },
+      "safari": {
+        "supported": 1,
+        "minVersion": 9.1,
+		"details": [
```
Comment (cvan) [inline src/data.json]: can you use soft spaces instead of tabs (so we're consistent with the formatting elsewhere)?

File: src/data.json (hunk 3 above)
Comment (neuhaus) [inline src/data.json]: done

File: src/data.json (hunk 1 above)
Comment (neuhaus) [inline src/data.json]: done

File: src/data.json (hunk 4 above)
Comment (neuhaus) [inline src/data.json]: done

File: src/data.json (hunk 5 above)
Comment (neuhaus) [inline src/data.json]: done

File: src/data.json [hunk 6]
```diff
@@ -323,6 +323,41 @@
...
+		  "requires <code>webkit</code> prefix"
```
Comment (cvan) [inline src/data.json]: It's probably worth including the "known issue" listed under the [`Can I Use?` entry for the `Web Audio API`](http://caniuse.com/#feat=audio-api):

> Safari does not appear to support `createMediaElementSource`.


<hr>

Can you please change this line to the following?

```json
  "Requires <code>-webkit</code> prefix. Known issue: <a href=\"https://developer.mozilla.org/en-US/docs/Web/API/AudioContext/createMediaElementSource\"><code>createMediaElementSource</code></a> <a href=\"https://bugs.webkit.org/show_bug.cgi?id=125503#c3\">is currently unsupported in WebKit/Safari</a>."
```

Thanks so much!

File: src/data.json [hunk 7]
```diff
@@ -323,6 +323,41 @@
...
+		]
+      },
+      "edge": {
+        "supported": 1,
+        "minVersion": 13
+      },
+      "samsung": {
+        "supported": 1,
+		"details": [
+		  "<a href=\"http://developer.samsung.com/internet#gearvr-overview\">source</a>"
```
Comment (cvan) [inline src/data.json]: Can you change this line to the following?

```json
"Fully supported. See <a href=\"http://developer.samsung.com/internet#gearvr-overview\">support info.</a>"
```

**Comment by cvan** [comment]
Thanks for adding the Web Audio API. It was definitely an oversight on my part. Really awesome work here - it's much appreciated.

Let me know when you've addressed all the comments I suggested. Thanks again!

File: src/data.json (hunk 7 above)
Comment (neuhaus) [inline src/data.json]: done

**Comment by neuhaus** [comment]
I think i fixed all the issues. Thanks for the review.

File: src/data.json (hunk 6 above)
Comment (neuhaus) [inline src/data.json]: done

**Comment by neuhaus** [comment]
OK, I missed one, but that's also fixed now

File: src/data.json [hunk 8]
```diff
@@ -353,7 +353,7 @@
       "samsung": {
         "supported": 1,
         "details": [
-          "<a href=\"http://developer.samsung.com/internet#gearvr-overview\">source</a>"
+          "Fully supported. See <a href=\"http://developer.samsung.com/internet#gearvr-overview\">support info.</a>"
```
Comment (cvan) [inline src/data.json]: slight nitpick: could you move the period `.` outside the `</a>`?

Comment (neuhaus) [inline src/data.json]: done. I like your attention to detail.

File: src/data.json [hunk 9]
```diff
@@ -323,6 +323,39 @@
       "samsung": {
         "supported": 0
       }
+    },
+    {
+      "name": "Web Audio API",
+      "description": "Positional audio is essential for VR.<br><br><a href=\"https://webaudio.github.io/web-audio-api/#Spatialization\">Spec</a>.",
```
Comment (cvan) [inline src/data.json]: Can you rephrase this to point to https://webaudio.github.io/web-audio-api/ instead of https://webaudio.github.io/web-audio-api/#Spatialization:

In particular:

```json
  "description": "The <a href=\"https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API\">Web Audio API</a> enables manipulation of audio, ability to choose audio sources, add effects to audio, create audio visualizations, apply spatial effects (e.g.,, panning) , and etc.",
```
